packages/xml-views-completion/src/providers/attributeValue/literal.ts

```diff
@@ -0,0 +1,51 @@
+import { map, find } from "lodash";
+import { XMLAttribute } from "@xml-tools/ast";
+import { flattenProperties } from "@ui5-language-assistant/logic-utils";
+import { BooleanValueInXMLAttributeValueCompletion } from "../../../api";
+import {
+  getClassByElement,
+  filterMembersForSuggestion
+} from "../utils/filter-members";
+import { UI5AttributeValueCompletionOptions } from "./index";
+
+/**
+ * Suggests boolean value (true/false) inside attribute value
+ */
+export function booleanSuggestions(
+  opts: UI5AttributeValueCompletionOptions
+): BooleanValueInXMLAttributeValueCompletion[] {
+  const xmlElement = opts.element;
+  const xmlAttribute = opts.attribute;
+
+  const elementClass = getClassByElement(xmlElement, opts.context);
+  if (elementClass === undefined) {
+    return [];
+  }
+  const properties = flattenProperties(elementClass);
+  const ui5Property = find(properties, ["name", xmlAttribute.key]);
+  const propType = ui5Property?.type;
+  if (propType?.kind !== "PrimitiveType" || propType.name !== "Boolean") {
+    return [];
+  }
+
+  const values = [
+    {
+      name: "true",
+      value: true
+    },
+    {
+      name: "false",
+      value: false
+    }
+  ];
+  const prefix = opts.prefix ?? "";
+  const prefixMatchingValues = filterMembersForSuggestion(values, prefix, []);
```

Comment:
would this work for single quotes?

Reply:
Do we need the filter utility or could we use a regExp to check? which would be simpler?

Reply:
We need to filter by the prefix, and I prefer to do it with the same utility function in case we change the logic in the future (e.g. to be more vscode-like - all characters included instead of substring)